docs/tutorial.rst

```diff
@@ -289,116 +375,78 @@ This is useful for integrating with tools such as
+    node 10: time = 5379.737460469677
 
-.. code-block:: python
 
-    import msprime
-    import numpy as np
+In this code chunk we iterate up the tree starting at node 0 and
+stop when we get to the root. We know that a node is a root
+if its parent is :const:`tskit.NULL`, which is a special
+reserved node. (The value of the null node is -1, but we recommend
+using the symbolic constant to make code more readable.) We also use
+the :meth:`~tskit.Tree.time` method to get the time
+for each node, which corresponds to the time in generations
+at which the coalescence event happened during the simulation.
+We can also obtain the length of a branch joining a node to
+its parent using the :meth:`~tskit.Tree.branch_length`
+method::
 
-    def segregating_sites_example(n, theta, num_replicates):
-        S = np.zeros(num_replicates)
-        replicates = msprime.simulate(
-            Ne=0.5,
-            sample_size=n,
-            mutation_rate=theta / 2,
-            num_replicates=num_replicates)
-        for j, tree_sequence in enumerate(replicates):
-            S[j] = tree_sequence.num_sites
-        # Now, calculate the analytical predictions
-        S_mean_a = np.sum(1 / np.arange(1, n)) * theta
-        S_var_a = (
-            theta * np.sum(1 / np.arange(1, n)) +
-            theta**2 * np.sum(1 / np.arange(1, n)**2))
-        print("              mean              variance")
-        print("Observed      {}\t\t{}".format(np.mean(S), np.var(S)))
-        print("Analytical    {:.5f}\t\t{:.5f}".format(S_mean_a, S_var_a))
+    >>> print(tree.branch_length(6))
+    117.98558842923524
 
-Running this code, we get::
+The branch length for node 6 is about 118 generations, since
+the birth times of node 6 was 11 generations ago, and the birth time of its
+parent, node 7, was around 129 generations ago.
+It is also
+often useful to obtain the total branch length of the tree, i.e.,
+the sum of the lengths of all branches::
 
-    >>> segregating_sites_example(10, 5, 100000)
-              mean              variance
-    Observed      14.17893          53.0746740551
-    Analytical    14.14484          52.63903
+    >>> print(tree.total_branch_length)
+    13238.125493096279
 
+.. _sec_demography:
 
-Note that in this example we set :math:`N_e = 0.5` and
-the mutation rate to :math:`\theta / 2` when calling :func:`.simulate`.
-This works because ``msprime`` simulates Kingman's coalescent,
-for which :math:`N_e` is only a time scaling;
-since :math:`N_e` is the diploid effective population size,
-setting :math:`N_e = 0.5` means that the mean time for two samples to coalesce
-is equal to one time unit in the resulting trees.
-This is helpful for converting the diploid per-generation time units
-of msprime into the haploid coalescent units used in many
-theoretical results. However, it is important to note that conventions
-vary widely, and great care is needed with such factor-of-two
-rescalings.
+Demography
+**********
```

Comment:
Not necessarily for this PR, but this Demography section feels pretty advanced; a walk-through of the steps in the `migration_example( )` in words would be good, maybe.

Reply:
Yes -- I have flagged this in a previous issue, and have simplified content ready to port in from elsewhere. I will do this once I've finished the 'Getting started' section!

Reply:
see #808

Reply:
ah whoops